sunpy/net/dataretriever/sources/goes.py

```diff
@@ -17,7 +19,32 @@
 
 TIME_FORMAT = config.get("general", "time_format")
 
-__all__ = ['XRSClient']
+__all__ = ["XRSClient", "SUVIClient"]
+
+import urllib.request
+import requests
+from bs4 import BeautifulSoup
+
+def url_is_alive(url):
```

Comment:
I am pretty sure we already have a function to do this in `util.net`

Reply:
On a large side note: We should use that to create a test that reports what servers are down or up that we access for our tests and examples.

Reply:
:thinking: nice idea.

Reply:
Opened an issue: https://github.com/sunpy/sunpy/issues/3303

Reply:
I'm pretty sure we do. I was just trying to work fast here. I'll get rid of this function which is currently not even used anyway.